src/containers/Landing/index.tsx

```diff
@@ -0,0 +1,19 @@
+import { styled } from '@mui/material/styles';
+
+export const Landing = () => {
+  return (
+    <LandingContainer>
+      <h1>Web3 React Boilerplate</h1>
+    </LandingContainer>
+  );
+};
+
+const LandingContainer = styled('div')({
+  display: 'flex',
+  flexDirection: 'column',
+  height: 'calc(100vh - 16rem)',
```

Comment:
don't like the idea of hardcoding this here

Reply:
https://github.com/defi-wonderland/web3-react-boilerplate/pull/8/commits/f97a49e8d1b84e01dea3db2ef87ddd6541413e2e